lib/text/vtt_text_parser.js

```diff
@@ -955,10 +955,13 @@ shaka.text.VttTextParser = class {
       '&gt;': '>',
       '&quot;': '"',
       '&#39;': '\'',
+      '&nbsp;': ' ',
```

Comment:
To preserve the non-breaking space, use '\u{a0}'

Reply:
Done!